Resources/doc/index.md

```diff
@@ -147,3 +147,30 @@ class BotController extends Controller
 
 }
 ```
+
+### Test user agents
+
+The bundle provides a console command to test user agents, you can run it by calling:
+
+```
+php app/console bot-detect:test-user-agent "AddThis"
```

Comment:
Can you add a `$` to the start of the line? Maybe it is also possible to mark this code block to be containing a bash command line. E.g.:

```
    ``` bash
    php app/console bot-detect:test-user-agent "AddThis"
    ```
```